.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       - name: Run clippy
         with:
           command: clippy
+           uses: actions-rs/cargo@v1
```

Comment:
Shouldn't this go above the `with` statement?

Reply:
You're completely right!! I force-pushed!